ComponentKit/StatefulViews/CKStatefulViewReusePool.h

```diff
@@ -20,6 +22,7 @@
 
 - (void)enqueueStatefulView:(UIView *)view
          forControllerClass:(Class)controllerClass
-                    context:(id)context;
+                    context:(id)context
+            enqueueComplete:(CKStatefulViewReusePoolPendingMayRelinquishBlock)enqueueComplete;
```

Comment:
I'd name the parameter to better match how the block is being used. "Enqueue complete" sounds more informative than instructive. Maybe something that better captures the spirit of how the block is being used?